variables.tf

```diff
@@ -308,3 +308,225 @@ variable "check_ec2_imdsv2" {
+}
+
+variable "check_ecs_no_environment_secrets" {
+  description = "Enable ecs-no-environment-secrets rule"
+  type        = bool
+  default     = true
+}
+
+variable "ecs_no_environment_secrets" {
+  description = "Comma-separated list of key names to search for in the environment variables of container definitions within Task Definitions. Extra spaces will be removed."
+  type        = string
+  default     = "keys,csv"
+}
+
+variable "enable_efs_encrypted_check" {
+  description = "Enable efs-encrypted-check rule"
+  type        = bool
+  default     = true
+}
+
+variable "kms_key_id" {
+  description = "Amazon Resource Name (ARN) of the KMS key that is used to encrypt the EFS file system."
+  type        = string
+  default     = "keys,csv"
+}
+
+variable "check_elb_deletion_protection_enabled" {
+  description = "Enable elb-deletion-protection-enabled rule"
+  type        = bool
+  default     = true
+}
+
+variable "check_elb_logging_enabled" {
+  description = "Enable elb-logging-enabled rule"
+  type        = bool
+  default     = false
+}
+
+variable "elb_logging_s3_buckets" {
+  description = "Comma-separated list of Amazon S3 bucket names for Amazon ELB to deliver the log files."
+  type        = string
+  default     = ""
+}
+
+variable "check_iam_policy_no_statements_with_admin_access" {
+  description = "Enable iam-policy-no-statements-with-admin-access rule"
+  type        = bool
+  default     = true
+}
+
+variable "check_iam_policy_no_statements_with_full_access" {
+  description = "Enable iam-policy-no-statements-with-full-access rule"
+  type        = bool
+  default     = true
+}
+
+variable "exclude_permission_boundary" {
+  description = "Boolean to exclude the evaluation of IAM policies used as permissions boundaries. If set to 'true', the rule will not include permissions boundaries in the evaluation. Otherwise, all IAM policies in scope are evaluated when set to 'false.'"
+  type        = bool
+  default     = false
+}
+
+variable "check_nacl_no_unrestricted_ssh_rdp" {
+  description = "Enable nacl-no-unrestricted-ssh-rdp rule"
+  type        = bool
+  default     = true
+}
+
+variable "check_internet_gateway_authorized_vpc_only" {
+  description = "Enable internet-gateway-authorized-vpc-only rule"
+  type        = bool
+  default     = false
+}
+
+variable "authorized_vpc_ids" {
+  description = "Comma-separated list of the authorized VPC IDs with attached IGWs. If parameter is not provided all attached IGWs will be NON_COMPLIANT."
+  type        = string
+  default     = "keys,csv"
+}
+
+variable "check_rds_snapshot_encrypted" {
+  description = "Enable rds-snapshot-encrypted rule"
+  type        = bool
+  default     = true
+}
+
+variable "check_rds_cluster_deletion_protection_enabled" {
+  description = "Enable rds-cluster-deletion-protection-enabled rule"
+  type        = bool
+  default     = true
+}
+
+variable "check_db_instance_backup_enabled" {
+  description = "Enable db-instance-backup-enabled rule"
```

Comment:
The description says `Enable db-instance...` but the default value is `false`. Quite a few of these say `Enable ...` but their values are `false`. It's `Enable <rule> rule` but other descriptions clearly explain what the config rule is for or does. The language on these should be consistent.

Reply:
There are two types of variables in aws config in this file. The first is just whether or not the rule is on, and the second type is a parameter that is either required or optional for that rule. The parameters are the variables with the more in-depth descriptions. Since I wasn't changing any other aspects of the module/rules, I just modeled after what is already there. In this instance, I chose default as false since it deals with a RDS backup and not everyone might be using RDS. On all of the new rules I added, anything that has to do with a backup is default false since that can be a cost constraint for storage, etc.